### PR TITLE
Remove embedding docs

### PR DESCRIPTION
**Pull request by seanh**
These have moved to the client docs:
http://h.readthedocs.io/projects/client/en/latest/publishers/embedding/

I'll add a readthedocs page redirect from the deleted URL to the new
one: http://docs.readthedocs.io/en/latest/user-defined-redirects.html#page-redirects

I've also deleted an old embedding.rst stub page which was just a link
to the (now also deleted) publishers/embedding.rst page, leftover from
the last time this page got moved. I'll also add a readthedocs page
redirect for this one.

### CODE REVIEW

**Comment by seanh** [comment]
I've added redirects in the h readthedocs project's admin, so http://h.readthedocs.io/en/latest/publishers/embedding/ and http://h.readthedocs.io/en/latest/embedding/ should now both redirect to the client docs.

**Comment by fatbusinessman** [comment]
> http://h.readthedocs.io/en/latest/publishers/embedding/ and http://h.readthedocs.io/en/latest/embedding/ should now both redirect to the client docs.

Do you mean they should be redirecting _now_, or they should redirect once this PR gets merged? Because they don’t seem to be redirecting yet.

**Comment by seanh** [comment]
They should be redirect now, I'm not sure why they aren't, let me try again

**Comment by seanh** [comment]
Ok, I think the redirects will only start working after this is merged: http://docs.readthedocs.io/en/latest/user-defined-redirects.html#implementation

**Comment by fatbusinessman** [comment]
Yup, looks like you were right. I just checked both these URLs post-merge, and they both redirect to suitable places in the client docs.

Nice work finding out this was an option. 👍
